src/AaveAvalancheTreasuryMigration_20230903/MIGRATE_AVALANCHE_V2_TREASURY_TO_AVALANCHE_V3.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Treasury Management - Avalanche v2 to v3 Migration
+discussions: https://governance.aave.com/t/arfc-treasury-management-avalanche-v2-to-v3-migration/14469
+author: TokenLogic
+---
+
+# Summary
+
+Redeem the Aave DAO funds on Aave v2 Avalanche and deposit underlying assets in Aave v3 Avalanche.
+
+# Motivation
+
+Aave DAO currently holds around $4.52M on Avalanche v2. There are several drivers for migrating Aave DAO's funds:
+
+* Higher deposit rates
+* wAVAX incentives
+* Encourage migration v2 to v3
+
+Most of these funds can be migrated to v3 with the exception of USDT. There is not enough liquidity to redeem the entire 541.51K of USDT. As a result, approximately 25% of the USDT will be redeemed and deposited into v3.
```

Comment:
You mention migrating USDT but it's not in the specification below

Reply:
Seems USDT was removed entirely from the AIP so worth removing it from here unless I'm missing something